share/spice/forecast/forecast.css

```diff
@@ -274,3 +272,20 @@
 #fe_temp_switch .is-active {
 	font-weight: bold;
 }
+
+/*
+    Dark mode specific changes
+*/
+
+.dark-bg .fe_label a:hover {
+    color: #dadada;
```

Comment:
We shouldn't hard-code this, we need to fix this at a higher level as it affects all IA links in dark mode -- I'll open a task for the team.